lib/QuipProcessor.js

```diff
@@ -295,8 +295,26 @@ class QuipProcessor {
                 dateStr = updatedDate === createdDate? createdDate : `${createdDate} (Updated: ${updatedDate})`;
                 dateStr = `, ${dateStr}`;
             }
+
+            let highlightedSections = "";
+            if (
+                message.annotation &&
+                message.annotation.id &&
+                message.annotation.highlight_section_ids
+            ) {
+                highlightedSections = message.annotation.highlight_section_ids.map((sectionId, index) => {
+                    // Not sure if there can be more than one highlight section,
+                    // but if so, we can number the reference links
+                    const suffix =
+                        message.annotation.highlight_section_ids.length == 1
+                        ? ""
+                        : ` ${index + 1}`;
+                    return `<a href="#${sectionId}">ref${suffix}</a>`;
+                })
+                .join(" ");
+            }
             if(text) {
-                html += `\n<div class="message"><span class="message--user">${message.author_name}${dateStr}<br/></span>${text}</div>`;
+                html += `\n<div class="message"><span class="message--user">${message.author_name}${dateStr} ${highlightedSections}<br/></span>${text}</div>`;
```

Comment:
Wondering now if I should remove the space between `${dateStr}` and `${highlightedSections}` but I think it shouldn't matter since HTML essentially ignores whitespace between tags. But if necessary I can just incorporate it into the `highlightedSections` variable.